app/models/appointment.rb

```diff
@@ -0,0 +1,7 @@
+class Appointment < ApplicationRecord
+  validates :professional, presence: true
+  validates :date_time, presence: true
+  validates :specialty, presence: true
+  validates :diagnostics, presence: true
+  validates :complaints, presence: true
```

Comment:
Da pra fazer  tudo isso no mesmo validates :nail_care: 
```suggestion
  validates :professional, :date_time, :specialty, :diagnostics, :complaints, presence: true
```